packages/react/src/components/input/input.tsx

```diff
@@ -48,9 +46,8 @@ Input.displayName = 'Input';
 
 export type InputProps = BaseProps &
   Omit<InputElementProps, 'children'> & {
-    /** @deprecated Use `label` prop instead */
+    /** @deprecated Use component composition instead */
```

Comment:
Can we add an example here, or a link to storybook?
Not everyone might know what we mean by "component composition".

Reply:
Added examples, altought not their nor deprecation message is shown on editor, it just says "deprecated"...

Reply:
That's fine, it'll be hard to see the deprecation notice in the IDE regardless, because of how JSX children are shown.

We'll have to do a good job at notifying this change.